src/graphgl/queries/project.js

```diff
@@ -0,0 +1,24 @@
+import { gql } from "@apollo/client";
+
+export default gql`
+  query ($id: ID!) {
```

Comment:
better specify name on operation, it is not necessary, but comfortable to find operation in inspector with name
![image](https://github.com/Raily01/task_tracker_react/assets/51396910/2ddeb78d-742a-4fae-afea-2524e1fe4cd8)

```suggestion
  query Project($id: ID!) {
```